packages/main/src/SliderBase.ts

```diff
@@ -108,6 +109,14 @@ abstract class SliderBase extends UI5Element {
 	@property({ type: Boolean })
 	showTooltip = false;
 
+	/**
+	 * Enables handle tooltip displaying the current value.
+	 * @default false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	editableTooltip = false;
```

Comment:
false is obsolete:
https://sap.github.io/ui5-webcomponents/docs/development/properties/

Reply:
This is no longer the case, since the introduction of the property initializers - https://github.com/SAP/ui5-webcomponents/pull/8846

Reply:
Are you sure? I checked the documentation of the PR and for Boolean it says:

> initialize to false (not strictly necessary, but will infer the type boolean and is less repetition)

What is more the nightly documentation still has the sentence I quoted:
https://sap.github.io/ui5-webcomponents/nightly/docs/development/properties/

> Note: due to how HTML attributes work, boolean properties can only have false as a default value.

@pskelin Should this documentation be adjusted or this is still true?

Reply:
In short, the change should stay and the docs are correct, here is a bigger explanation:

### 1. booleans can only be default false
The first point is that default values for booleans can only be false, meaning you should not set `enableTooltip = true` (there will be a build error, but leaving out the attribute would make the property true and at the same time attribute is missing and this means false). To avoid such ambiguity, booleans have always been initialized to false to reflect how attributes work.

### 2. TS type
The part from the documentation is about typescript knowing the type of the field. There are two ways to achieve the same:
```
@property({ type: Boolean }) // the framework knows it's boolean
editableTooltip = false;    // TS knows it's boolean from the assigned value
```

second way is still correct but has more repetition
```
@property({ type: Boolean }) // used by the framework for attribute conversion
editableTooltip?: boolean;    // undefined will still evaluate to false, but boolean has to be repeated for TS
```

Reply:
Thanks!